### ISSUE TITLE
LateX not building in RST

### ISSUE THREAD

**Issue by jzf2101** [choldgraf/scipy_proceedings#17]
See section on binder costs @choldgraf

**Comment by choldgraf** [comment]
Yeah it looks like it's because @betatim 's edits used latex delimiters (`$`) while the rST needs `:math:` prefixes instead. I tried just replacing the `$` with `:math:` and that resulted in some rST errors. If anybody has time to debug this it'd be great!

**Comment by jzf2101** [comment]
I can debug this- I don't have rights to assign myself could you give me rights please @choldgraf

**Comment by choldgraf** [comment]
I already did a while ago - it's awaiting your response. Here's the link:

https://github.com/choldgraf/scipy_proceedings/invitations
